src/sentry/integrations/vercel/integration.py

```diff
@@ -164,15 +171,16 @@ def get_organization_config(self):
             ),
         )
 
-        fields = [
+    def get_organization_config(self):
+        return [
             {
                 "name": "project_mappings",
                 "type": "project_mapper",
                 "mappedDropdown": {
-                    "items": vercel_projects,
+                    "items": self._get_vercel_projects(),
                     "placeholder": _("Vercel project..."),
                 },
-                "sentryProjects": sentry_projects,
+                "sentry_projects": self._get_sentry_projects(),
```

Comment:
@mgaeta why is this changing to snake case?

Reply:
Good catch, reverted.